mods/lord/Blocks/lord_gems/aliases.lua

```diff
@@ -0,0 +1,6 @@
+-- @tags: legacy
+
+minetest.register_alias("lottores:blue_gem", "lord_gems:blue")
```

Comment:
используем одинарные кавычки
```suggestion
minetest.register_alias('lottores:blue_gem', 'lord_gems:blue')
```
и ниже